frontend/src/components/EventGroup.jsx

```
@@ -0,0 +1,62 @@
+        {events.map(event => (
+          <EventCard key={event.id} event={event} />
+        ))}
+      </div>
+
+      {hasMore && (
+        <div className="mt-4 text-center">
+          <button
+            onClick={fetchEvents}
+            className="text-blue-500 hover:underline font-medium"
+            disabled={loading}
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)